reconcile/gitlab_housekeeping.py

```
@@ -1452,12 +1453,16 @@ def run_error_healthcheck(
         has_rebase_error = REBASE_ERROR in labels
         mr_merge_error = getattr(mr, "merge_error", None)
-        mr_detailed = getattr(mr, "detailed_merge_status", None)
-        rebase_failed = bool(
-            mr_merge_error
-            and "Rebase failed" in mr_merge_error
-            and mr_detailed == "need_rebase"
-        )
+        if mr_merge_error and "Rebase failed" in mr_merge_error:
+            try:
+                fresh = gl.project.mergerequests.get(mr.iid)
+            except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)